Resources/config/forms/task_details.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" ?>
+<form xmlns="http://schemas.sulu.io/template/template"
```

Comment:
okay im confused now. why do we need this if we have a `FormMetadataLoader`? 😂

Reply:
we don't need it, before I implemented the FormMetadataLoader I tried something else with the xml template, but I kinda forgot to delete it afterwards 😅